lib/models/application.coffee

```diff
@@ -446,6 +446,95 @@ getApplicationModel = (deps, opts) ->
 		.asCallback(callback)
 
 	###*
+	# @summary Purge devices by application id
+	# @name purge
+	# @public
+	# @function
+	# @memberof resin.models.application
+	#
+	# @param {Number} appId - application id
+	# @returns {Promise}
+	#
+	# @example
+	# resin.models.application.purge(123);
+	#
+	###
+	exports.purge = (appId) ->
+		return request.send
+			method: 'POST'
+			url: '/supervisor/v1/purge'
+			baseUrl: apiUrl
+			body:
+				appId: appId
+				data:
+					appId: "#{appId}"
+		.catch (err) ->
+			if err.statusCode == LOCKED_STATUS_CODE
```

Comment:
This is undefined at the moment. Can you add the `LOCKED_STATUS_CODE` in `util` so it can be shared between `device.coffee` and `application.coffee`?

Reply:
where exactly? The util.coffee looks almost empty

Reply:
Do you mean this file --> https://github.com/resin-io/resin-sdk/blob/master/tests/util.coffee
Thats a file for test utilities.
You should put it in here I think --> https://github.com/resin-io/resin-sdk/blob/master/lib/util/index.coffee